CITATION.cff

```diff
@@ -0,0 +1,17 @@
+cff-version: 1.2.0
+title: >-
+  Substrait: Cross-Language Serialization for Relational
+  Algebra
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: substrait-io
+identifiers:
+  - type: url
+    value: 'https://github.com/substrait-io/substrait'
+repository-code: 'https://github.com/substrait-io/substrait'
+url: 'https://substrait.io/'
+license: Apache-2.0
+date-released: '2021-09-01'
```

Comment:
Why is it September 1st in this file and August 31st in the other file?  Can we have these match?

Reply:
Good question. Date in the year field is when the framework was found. 

Date released is the initial release date. Should we consider the initial release date for the year attribute too?

Reply:
I understand.